### PR TITLE
app-backup/rear: version bump to 2.6

### PR DESCRIPTION
**Pull request by madmartin**
re-introduce udev rule for backup start on backup disc connect
add proxy-maintainership

Closes: https://bugs.gentoo.org/696678
Package-Manager: Portage-3.0.4, Repoman-3.0.1
Signed-off-by: Martin Dummer <martin.dummer@gmx.net>

### CODE REVIEW

File: app-backup/rear/rear-2.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	sys-apps/iproute2
+	sys-apps/lsb-release
+	sys-apps/sed
+	sys-apps/util-linux
+	sys-block/parted
+	sys-boot/syslinux
+	virtual/cdrtools
+	!libressl? ( dev-libs/openssl:0= )
+	libressl? ( dev-libs/libressl:0= )
+	samba? ( net-fs/cifs-utils )
```
Comment (juippis) [inline app-backup/rear/rear-2.6.ebuild:31]: This USE flag doesn't seem to have any function in this ebuild. Can you move it to use *optfeature.eclass* instead?

File: app-backup/rear/rear-2.6.ebuild [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit udev
+
+DESCRIPTION="Relax-and-Recover is a setup-and-forget bare metal disaster recovery solution"
+HOMEPAGE="http://relax-and-recover.org/"
```
Comment (juippis) [inline app-backup/rear/rear-2.6.ebuild:9]: Can you add the Github page here as 2ndary HOMEPAGE?

Comment (madmartin) [reply app-backup/rear/rear-2.6.ebuild]: @juippis yes, both done

**Comment by madmartin** [comment]
@juippis qa scan complete, you may review again.

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**